lib/core-integration/src/Test/Integration/Scenario/API/Shelley/TransactionsNew.hs

```diff
@@ -202,45 +206,55 @@ spec = describe "NEW_SHELLEY_TRANSACTIONS" $ do
         --       make sure wa wallet balance is increased by withdrawalAmt - fee
 
     it "TRANS_NEW_CREATE_04a - Single Output Transaction" $ \ctx -> runResourceT $ do
-
-        liftIO $ pendingWith "Missing outputs on response - to be fixed in ADP-985"
-
+        -- constructing Tx
         let initialAmt = 3*minUTxOValue
         wa <- fixtureWalletWith @n ctx [initialAmt]
         wb <- emptyWallet ctx
         let amt = (minUTxOValue :: Natural)
 
-        payload <- liftIO $ mkTxPayload ctx wb amt
+        (destAddr, constrPayload) <- liftIO $ mkTxPayload ctx wb amt
 
         (_, ApiFee (Quantity feeMin) _ _ _) <- unsafeRequest ctx
-            (Link.getTransactionFee @'Shelley wa) payload
-        rTx <- request @(ApiConstructTransaction n) ctx
-            (Link.createUnsignedTransaction @'Shelley wa) Default payload
-        verify rTx
+
+            (Link.getTransactionFee @'Shelley wa) constrPayload
+        rConstrTx <- request @(ApiConstructTransaction n) ctx
+            (Link.createUnsignedTransaction @'Shelley wa) Default constrPayload
+        verify rConstrTx
             [ expectSuccess
             , expectResponseCode HTTP.status202
-            , expectField (#coinSelection . #inputs) (`shouldSatisfy` (not . null))
-            , expectField (#coinSelection . #outputs) (`shouldSatisfy` (not . null))
```

Comment:
Why removing this assertion, especially that the test was failing before due to missing outputs? (pls see `pendingWith` message)

Reply:
just check it later in the same test, but deeper:
```
        let coinSelOutputs = getFromResponse (#coinSelection . #outputs) rConstrTx
        coinSelOutputs `shouldBe`
            [ApiCoinSelectionOutput destAddr (Quantity amt) (ApiT W.empty)]
```